### DISCUSSION TITLE
Are there any GAIA-X extensions or samples?

### DISCUSSION THREAD

**Discussion by rawmahn** [eclipse-edc/Connector#1139] [Q&A]
Hello Community!

Are there any samples/docs/repos of GAIA-X extensions for EDC? In particular for GAIA-X - specific policies, self-descriptions, contract negotiation rules? Any other insights with respect to EDC's support of GAIA-X?

**Comment by sebbader** [comment]
Hi @rawmahn, not too much, just a few things I heard from Gaia-X implementation projects. There shall be a Java library for the Gaia-X information model. Policies, self-descriptions, contract negotiations etc. will however look very similar to the IDS specifications. So I personally expect a little different flavour but more or less similar requirements.
